KsApi/MockService.swift

```diff
@@ -278,7 +278,7 @@ internal struct MockService: ServiceType {
 
     self.publishUpdateError = publishUpdateError
 
-    self.fetchMessageThreadResponse = fetchMessageThreadResponse ?? .template
+    self.fetchMessageThreadResponse = fetchMessageThreadResponse
```

Comment:
Ah yes, this case is a little different because the `fetchMessageThreadResponse` is actually of type `MessageThread` and not the entire `MessageThreadEnvelope?`. It certainly could be the entire thing, I'm not sure why it was originally set up this way. You have pointed out that the error is not being tested here so I can add that in. Right now, the "error" that's being tested is the empty messages, which is determined by the nil `fetchMessageThreadResponse`.